### ISSUE TITLE
[Marketplace] Can't check out using Stripe

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2615]
### Expected behavior
After submitting payment, should place order and move to completed page

### Actual Behavior
Checkout page just reloads

### Steps to Reproduce the Behavior
1. Login as admin and configure payment method to Stripe
1. Attempt to checkout
1. Observe that after you hit "Complete Your Order" that it will pause and then just return to the checkout page with no errors.

### Versions
```
Node: 7.10.0
NPM: 5.0.3
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.11.0
Reaction: 1.5.0
Reaction branch: marketplace
Docker: 17.06.0-ce
```

**Comment by spencern** [comment]
Assigning myself to this for now as I think it's related to some legacy marketplace code. Hope to get it fixed during my stripe connect build.

**Comment by spencern** [comment]
I think this issue will be solved by the rebuild of the `reaction-stripe` package in the `spencer-stripe-connect-merchant-signup` branch. Will PR that branch today and then split off a new branch for the checkout code.

**Comment by spencern** [comment]
Fixed by #2682
